### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ISamofalov**
- [DEMO LINK](https://isamofalov.github.io/layout_moyo-header)
- [TEST REPORT LINK](https://isamofalov.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,79 @@
...
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
```
Comment (vlad-demchuk) [inline src/style.css:38]: you can use not instead of reset for the first child

File: src/style.css [hunk 2]
```diff
@@ -1 +1,79 @@
...
+.nav__link.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+  height: 4px;
+  border-radius: 2px;
+  background-color: #00acdc;
```
Comment (vlad-demchuk) [inline src/style.css:78]: move color to variable

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
GJ

File: src/index.html [hunk 3]
```diff
@@ -2,13 +2,68 @@
...
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__image"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
```
Comment (volodymyr-soltys97) [inline src/index.html:29]: You can make the links more informative
```suggestion
            <a href="#apple" class="nav__link is-active">Apple</a>
```

Comment (ISamofalov) [reply src/index.html]: please check, i made changes

**Pull request closed without merging**